Rakefile

```diff
@@ -145,21 +54,8 @@ task :api_samples do
 end
 CLOBBER.include FileList[%w[_includes/api-sample]]
 
-desc "Dump macOS formulae and analytics data"
-task formula_and_analytics: %i[formulae analytics]
-
-desc "Dump macOS casks and analytics data"
-task cask_and_analytics: %i[cask analytics]
-
-desc "Dump Linux analytics data"
-task :linux_analytics do
-  Rake::Task["analytics"].tap(&:reenable).invoke("linux")
-end
-
-desc "Dump all analytics (macOS and Linux)"
-task all_analytics: :analytics do
-  Rake::Task["analytics"].tap(&:reenable).invoke("linux")
-end
+desc "Generate the API files"
+task generate: %i[formulae casks analytics api_samples]
 
 desc "Build the site"
 task :build do
```

Comment:
Not sure if we're using much of the stuff below? Feels like it could either get run automatically or removed.

Reply:
Yeah, it's just a question of whether having them in the `Rakefile` is useful for those running an instance locally for debugging purposes. The benefit as I see it is that it's all in one place.

I think we probably can remove the `Rakefile` totally if we wanted to and do everything in GitHub actions. It's a little harder to generate the site since you then have to know the specific commands to run to generate/build/serve but we can also document that somewhere.